cibuildwheel/windows.py

```diff
@@ -106,21 +89,19 @@ def shell(args, env=None, cwd=None):
     temp_dir = tempfile.mkdtemp(prefix='cibuildwheel')
     built_wheel_dir = os.path.join(temp_dir, 'built_wheel')
 
-    if IS_RUNNING_ON_TRAVIS:
-        # instal nuget as best way for provide python
-        shell(["choco", "install", "nuget.commandline"])
-        # get pip fo this installation which not have. 
-        get_pip_url = 'https://bootstrap.pypa.io/get-pip.py'
-        get_pip_script = 'C:\\get-pip.py'
-        shell(['curl', '-L', '-o', get_pip_script, get_pip_url])
+    # instal nuget as best way for provide python
+    nuget = 'C:\\nuget.exe'
+    download('https://dist.nuget.org/win-x86-commandline/latest/nuget.exe', nuget)
+    # get pip fo this installation which not have.
+    get_pip_script = 'C:\\get-pip.py'
+    download('https://bootstrap.pypa.io/get-pip.py', get_pip_script)
```

Comment:
Just curious: do nuget versions not have `pip` installed?

Reply:
I've not redone the test, it's merely a rewrite of the travis-ci PR which was doing this.

Reply:
I'm not sure we hit the conditional statement lower in the code.

Reply:
Hmmm, two lines and a download. Not sure if it's worth testing whether we can remove the lines.

Reply:
[Seems pip is not installed](https://travis-ci.org/mayeut/cibuildwheel/jobs/606729148?utm_medium=notification&utm_source=github_status)

> Hmmm, two lines and a download. Not sure if it's worth testing whether we can remove the lines.

Not sure I fully understand this comment. If it only refers to `get-pip` then no, we can't remove those (now tested).

Reply:
Yeah, that's what I meant. That's kind of silly. Why do all these distributions not have `pip` installed?